mapie/regression/regression.py

```diff
@@ -628,7 +628,7 @@ def predict(
 
             alpha_np = cast(NDArray, alpha)
             if not allow_infinite_bounds:
-                n = len(self.conformity_scores_)
+                n = np.sum(~np.isnan(self.conformity_scores_))
```

Comment:
Why do we only calculate from the scores that aren't `np.nan`?

Reply:
This is because the `split` method retains `nan` values for the data used to train the model. Because we need to keep the size of calibration dataset, we only sum those that have a defined conformity score.